error_dict = \
{'unbalanced-quotation': 'During processing, result ended up as:\n\n{}\n\nDid you try to manually type the file name?\n\nTry using numbers instead.',
 
 'command not recognized': "Neither the application, nor the shell understands the command '{}'.\n\nTry again.",
 
 'keyboard-interrupt': "Return to continue\n\nQuit or Q to quit at prompt.",

 'stdout': "Command {} resulted in the following output:\n\n{}",

 'stderr': "Command {} produced an error:\n\n{}",

 'trash confirm': "You are about to put the following file/folder(s) into the trash.\n\n {}\n\nAre you sure you want to do this?",

 'path does not exist': "The path:\n\n{}\n\ndoes not exist. Try again.",

 'out of range': "There is no item by the number: {}",
 'bad arguments': "An unexpected argument was passed to {}.\n\nHere is the documentation for the command:\n\n{}",
 'required argument missing': "The command {} requires an argument but was not given.\n\nHere is the documentation for the command:\n\n{}",
 'no saved': "Use the command 'remember' to use commands that use it.",
 'permission denied': "You do not have permission to access the item:\n\n{}",
 }
